evm/evmone/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ethereum/evmone
 RUN cd evmone && mkdir build
 RUN cd evmone/build && CXXFLAGS="-Wno-error=deprecated-copy" cmake .. -DEVMONE_TESTING=ON
```

Comment:
Is `CXXFLAGS="-Wno-error=deprecated-copy"` still needed?

Reply:
It is no longer needed, I've removed it.